### DISCUSSION TITLE
subdomain setup [Jan 02]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#4788] [Discord Threads]
> 🙋‍♂️ **vikiiingur**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### I've seen our users use it with subdomain. And I understand you are looking for nginx et al setup 🙂   
####    
#### There was a git discussion  
#  
> 💁 **o1lab**  
#### let me try to find it  
#  
> 💁 **o1lab**  
#### somewhat related (but at API level)  
#  
> 💁 **o1lab**  
#### https://github.com/nocodb/nocodb/issues/428  
#  
> 🙋‍♂️ **vikiiingur**  
#### hi, thanks for this, I came across of this page too, but it did not help  
#  
> 🙋‍♂️ **vikiiingur**  
#### my problem is more related to linking nocodb to subdirectory via nginx rather than subdomain  
#  
> 🙋‍♂️ **vikiiingur**  
#### subdomain works for me, but I cannot use it due to the specific setup of the server that I am not an admin of, I received a specific subdomain, under which I am able to issue multiple docker containers to subdirectories, e.g. aaa.doomain.tld/nocodb and aaa.domain.tld/postgresql  
#  
> 🙋‍♂️ **vikiiingur**  
#### I cannot make nocodb work this way with nginx proxy manager, I can make it work with the main aaa.domain.tld but not aaa.domain.tld/nocodb, which is not ideal as I will have more containers to run  
#  
> 🙋‍♂️ **vikiiingur**  
#### any help would be welcome  
#  
> 💁 **o1lab**  
#### You would have to fiddle two env variables. Backend_url and another similar.  
#  
> 🙋‍♂️ **vikiiingur**  
#### Yeah, I did, I believe all I have set correctly, as with the IPs I am able to reach the server, it is rather linking it up with nginx is the issue I have  
#  
> 🙋‍♂️ **vikiiingur**  
#### there is a lack of clear documentation on that unfortunately  
#  
> 💁 **o1lab**  
#### Would love a PR on it please (when sorted)  
#  
> 🙋‍♂️ **vikiiingur**  
#### added  
#  
> 🙋‍♂️ **Ariel**  
#### hi, I'm having the same issue with Apache instead of NGINX: I have a vainilla docker installation of nocodb running in the server and listening in 8080, but I need to access it externally via HTTPS to put it in an iframe of my secure website. To do so,  I've defined in apache a reverse proxy listening in a location (i.e. /nocodb) that proxies to localhost:8080  and as soon as I enter https://www.domain.com/nocodb in a browser, it redirects to https://www.domain.com/dashboard giving a 404.  Any ideas? this is driving me crazy. I am using the same proxy to access other services like Grafana and no problems. Thank you.  
#  
> 💁 **o1lab**  
#### Have u looked at all env variables  
#  
> 🙋‍♂️ **Ariel**  
#### If you are talking about https://docs.nocodb.com/getting-started/installation#environment-variables yes, I´ve tried everything. First of all, I´ve tried to get nocodb  to work on a subdirectory (.i.e http://www.domain.com/database/dashboard, setting NC_DASHBOARD_URL alone to "/database/dashboard"  instead of "/dashboard" but when I open a browser at http://www.domain.com/database/dashboard, the login screen shows but when I try to login, I get a 404 error trying to access  http://www.domain.com:8082/database/api/v1/db/meta/nocodb/info    
#### The documentation about env variables in the page is kind of lacky and it makes me guess which other env variable could be affecting a change on DASHBOARD_URL. I´ve tried setting NC_PUBLIC_URL, NUXT_PUBLIC_NC_BACKEND_URL, and NC_BASEURL_INTERNAL to different values with same result.  
#  
> 🙋‍♂️ **Ariel**  
#### no way to make it work in a subdirectory?  
#  
> 🙋‍♂️ **Ariel**  
#### #428 was never fixed?  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/938424910445944872)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
